lib/palette.js

```diff
@@ -98,6 +99,35 @@ function map_palette(a, b, type, bc)
 }
 
 /**
+* Returns the closest (or furthest) color to targetColor in palette, operating in the L,a,b colorspace for performance
+* @param {labcolor} targetColor should have fields L,a,b
+* @param [{labcolor}] palette each element should have fields L,a,b
+* @param 'findFurthest' should be falsy to find the closest color
+* @return {labcolor}
+*/
+function palette_lab(targetColor, palette, findFurthest)
```

Comment:
I originally implemented it in that way, but the jitter was much higher, I assume because of the object allocation (`c`) and the string construction (`palette_map_to_key`) was causing GC overhead.

Would you be happy if I made another method that implemented this behaviour on top of the current implementation, i.e.:

```js
function map_palette_lab(a, b, find_furthest) {
  return b.reduce(function(memo, color) {
    memo[lab_map_key(color)] = palette_lab(color, b, find_furthest);
    return memo;
  }, {});
}
```